documentation/source/user_section/tutorials/other/visualizing-misaligned-cords.rst

```diff
@@ -0,0 +1,28 @@
+.. TODO:
+
+   Is this one-page tutorial necessary? It is basically just telling users that the ``sct_flatten_sagittal`` tool exists. (Compared to other tutorials, which demonstrate multi-step workflows.)
+
+   So, I am thinking that maybe this page will be unnecessary once we organize the "Command-Line Tools" page into one-page-per-script. We could simply have all of this information on the dedicated "sct_flatten_sagittal" page instead, and save the "Tutorials" for complex workflows only.
+
+Visualizing misaligned cords with 2D sagittal flattening
+########################################################
+
+Because some subjects (esp. those with scoliosis) might not be perfectly aligned in the medial plane, it can sometimes be difficult to view the spinal cord in a single 2D slice. The function ``sct_flatten_sagittal`` applies slice-wise deformation to align the cord in the medial plane.
+
+.. code::
+
+   sct_flatten_sagittal -i t1.nii.gz -s t1_seg.nii.gz
```

Comment:
I think we need to provide `t1_seg.nii.gz` as a downloadable option because I dont have it currently. I only have `t1.nii.gz`. Also needs a line just before on where the data is.